tools/MongoToPostgresMigrator/docs/scripts/Initialize-MigrationTarget.ps1

```
@@ -0,0 +1,154 @@
+
+.EXAMPLE
+    # reset the target between migration attempts
+    .\Initialize-MigrationTarget.ps1 -PgHost localhost -Port 5433 -Username postgres -TruncateOnly
+#>
+[CmdletBinding()]
+param(
+    [Parameter(Mandatory)][string]$PgHost,
+    [int]$Port = 5432,
+    [string]$Database = 'featbit',
+    [Parameter(Mandatory)][string]$Username,
+    [string]$Password,
```

PSScriptAnalyzer (1.25.0)

[warning] Parameter '$Password' should not use String type but either SecureString or PSCredential, otherwise it increases the chance to expose this sensitive information.

Suggested fix: Set $Password type to SecureString; Set $Password type to PSCredential

(PSAvoidUsingPlainTextForPassword)